plugins/Vestige/Vestige.cpp

```diff
@@ -720,10 +720,9 @@ void VestigeInstrumentView::openPreset()
 
 	if ( m_vi->m_plugin != nullptr ) {
 		m_vi->m_plugin->openPreset();
-    		bool converted;
-    		QString str = m_vi->m_plugin->currentProgramName().section("/", 0, 0);
-     		if (str != "")
-   			lastPosInMenu = str.toInt(&converted, 10) - 1;
+		bool converted = false;
```

Comment:
So I should not initialize this one? I guess the same could be said elsewhere for when we pass the variable by pointer to initialize it? That would make sense because in that case, the value we initialize it with isn't really the initial value.

Reply:
I don't think these need to be default initialized.